providers/src/airflow/providers/google/cloud/operators/apache_kafka.py

```diff
@@ -451,3 +452,339 @@ def execute(self, context: Context):
+                metadata=self.metadata,
+            )
+            self.xcom_push(
+                context=context,
+                key="topic_page",
+                value=types.ListTopicsResponse.to_dict(topic_list_pager._response),
+            )
+        except Exception as error:
+            raise AirflowException(error)
+        return [types.Topic.to_dict(topic) for topic in topic_list_pager]
+
+
+class ApacheKafkaGetTopicOperator(ApacheKafkaBaseOperator):
+    """
+    Return the properties of a single topic.
+
+    :param project_id: Required. The ID of the Google Cloud project that the service belongs to.
+    :param location: Required. The ID of the Google Cloud region that the service belongs to.
+    :param cluster_id: Required. The ID of the cluster whose topic is to be returned.
+    :param topic_id: Required. The ID of the topic whose configuration to return.
+    :param retry: Designation of what errors, if any, should be retried.
+    :param timeout: The timeout for this request.
+    :param metadata: Strings which should be sent along with the request as metadata.
+    :param gcp_conn_id: The connection ID to use connecting to Google Cloud.
+    :param impersonation_chain: Optional service account to impersonate using short-term
+        credentials, or chained list of accounts required to get the access_token
+        of the last account in the list, which will be impersonated in the request.
+        If set as a string, the account must grant the originating account
+        the Service Account Token Creator IAM role.
+        If set as a sequence, the identities from the list must grant
+        Service Account Token Creator IAM role to the directly preceding identity, with first
+        account from the list granting this role to the originating account (templated).
+    """
+
+    template_fields: Sequence[str] = tuple(
+        {"cluster_id", "topic_id"} | set(ApacheKafkaBaseOperator.template_fields)
+    )
+    operator_extra_links = (ApacheKafkaTopicLink(),)
+
+    def __init__(
+        self,
+        cluster_id: str,
+        topic_id: str,
+        *args,
+        **kwargs,
+    ) -> None:
+        super().__init__(*args, **kwargs)
+        self.cluster_id = cluster_id
+        self.topic_id = topic_id
+
+    def execute(self, context: Context):
+        ApacheKafkaTopicLink.persist(
+            context=context,
+            task_instance=self,
+            cluster_id=self.cluster_id,
+            topic_id=self.topic_id,
+        )
+        self.log.info("Getting Topic: %s", self.topic_id)
+        try:
+            topic = self.hook.get_topic(
+                project_id=self.project_id,
+                location=self.location,
+                cluster_id=self.cluster_id,
+                topic_id=self.topic_id,
+                retry=self.retry,
+                timeout=self.timeout,
+                metadata=self.metadata,
+            )
+            self.log.info("Topic was gotten.")
```

Comment:
Maybe it makes sense to rephrase this message? :)

Reply:
@VladaZakharova I have updated :)